### PR TITLE
Add CNV activities

### PR DESCRIPTION
**Pull request by rawagner**
- Disk import
- v2v import

v2v_act

### CODE REVIEW

**Comment by andybraren** [comment]
@matthewcarleton @ranelim @yfrimanm ^ 🎉

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ActivityProgress } from '@console/shared/src/components/dashboard/activity-card/ActivityItem';
+import { ResourceLink } from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { VirtualMachineModel } from '../../../models';
+
+export const DiskImportActivity = ({ resource }) => (
```
Comment (mareklibra) [inline frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: please add type

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  >
+    <ResourceLink
+      kind={referenceForModel(VirtualMachineModel)}
+      name={resource.metadata.ownerReferences[0].name}
+      namespace={resource.metadata.namespace}
+    />
+  </ActivityProgress>
+);
+
+export const V2VImportActivity = ({ resource }) => (
```
Comment (mareklibra) [inline frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: likewise

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  <ActivityProgress
+    title="Importing VM (v2v)"
+    progress={parseInt(_.get(resource.metadata.annotations, 'v2vConversionProgress', 0), 10)}
```
Comment (mareklibra) [inline frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: can `resource` be undefined?

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  >
+    <ResourceLink
+      kind={referenceForModel(VirtualMachineModel)}
+      name={resource.metadata.ownerReferences[0].name}
```
Comment (mareklibra) [inline frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: Is presence of `ownerReference[0]` guaranteed? I understand how it is being created recently. But I would suggest to be more defensive.

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx (hunk 1 above)
Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: type added

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx (hunk 2 above)
Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: type added

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx (hunk 3 above)
Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: It cannot. This is a resource which is fetched from k8s (definition is https://github.com/openshift/console/pull/3577/files#diff-ebd2d0d608781a3c30e4c97813f813c2R284 ), run through `isActivity` for given activity like https://github.com/openshift/console/pull/3577/files#diff-ebd2d0d608781a3c30e4c97813f813c2R289 or https://github.com/openshift/console/pull/3577/files#diff-ebd2d0d608781a3c30e4c97813f813c2R306 which if returns true, then the resource is passed to component.

File: frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx (hunk 4 above)
Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/activity.tsx]: fixed. Im more defensive now

**Pull request merged**